cluster/Vagrantfile

```diff
@@ -141,6 +142,24 @@ if node_os == RHEL
   end
 end
 
+# show a warning that the FORWARD chain policy must be set to ACCEPT on Debian/Ubuntu
+# systems since they default to REJECT.  We can't check the actual value because root
+# access is required.
+if ARGV[0] == "up" && File.exists?("/etc/debian_version")
+  puts
+  puts "NOTE: Please ensure that the FORWARD chain's policy is set to ACCEPT"
+  puts "      by running: sudo iptables -P FORWARD ACCEPT"
+  puts
+end
+
+# if sysctl exists, ensure that ipv4 forwarding is enabled
+if find_executable0("sysctl") && %x{sysctl net.ipv4.ip_forward}.strip.chars.last == "0"
```

Comment:
`find_executable("foo")` prints "checking for foo... no" in addition to returning true/false

`find_executable0("foo")` just returns true/false

It looks like a typo 😄